mcs/class/corlib/System.Runtime.Remoting.Activation/ActivationServices.cs

```diff
@@ -88,6 +88,8 @@ public static IMessage RemoteActivate (IConstructionCallMessage ctorCall)
 
 		public static object CreateProxyFromAttributes (Type type, object[] activationAttributes)
 		{
+			if (!AppDomain.IsRemotingSupported)
```

Comment:
This does not need to be dynamic check

Reply:
What do you mean?

Reply:
I researched this and that method must go away.  However, since it is not called from the runtime, we can remove the conditional here and instead protect the call sites in `rttype.cs`.